Habinezajanvier/Capstone
# #23 user location

#### What does this PR do?
Getting user location using geolocation API
#### Description of Task to be completed?
- Getting longitude and latitude using geolocation API
- Getting real place using the third party API
#### How should this be manually tested?
- Clone this repo `git clone https://github.com/Habinezajanvier/my-brand-project.git`
- Switch the directory `cd my-brand-project`
- Checkout the branch `git checkout ft-user-location-#24`
- Using your preferred browser start `index.html` found in `/UI/folder
- Click on **blog** found in the navigation bar of the opened web
> You should see list of articles they are coming from firebase DB
- Click on any article of your choice in the article list
> You should see your own location in the top-right corner of the tab
#### Any background context you want to provide?
- Starting `index.html` using liveServer should be helpful
- Using Chrome as the preferred browser is highly recommended
#### What are the relevant pivotal tracker stories?
 Click [here](https://trello.com/c/V47dpU5M/23-getting-user-locations) for relevant trello story
